### ISSUE TITLE
torch::nn::ConvTranspose2dImpl's forward() method expects 2 arguments, but received 1 (forward at /usr/local/libtorch/include/torch/csrc/api/include/torch/nn/modules/container/any.h:384)"

### ISSUE THREAD

**Issue by guijiyang** [pytorch/pytorch#35469]
I use ConvTranspose2d in a Sequential, and forward called with one parameter, it invoke above issue. I think it is a bug of libtorch,  ConvTranspose2d need another forward overload function with one parameter.

cc @yf225

**Comment by mruberry** [comment]
Hi @guijiyang, this would require setting a default out_channels value, per the documentation here: https://pytorch.org/docs/master/nn.html?highlight=convtranspose2d#torch.nn.ConvTranspose2d. What would be a natural default value that wouldn't surprise users?

**Comment by guijiyang** [comment]
> Hi @guijiyang, this would require setting a default out_channels value, per the documentation here: https://pytorch.org/docs/master/nn.html?highlight=convtranspose2d#torch.nn.ConvTranspose2d. What would be a natural default value that wouldn't surprise users?

well I have set out_channels already:
` m_decoder = tnn::Sequential(
        tnn::ConvTranspose2d(tnn::ConvTranspose2dOptions(256, 128, 3).padding(1).bias(false)),
        tnn::InstanceNorm2d(128),
        tnn::ReLU(true),
        tnn::ConvTranspose2d(tnn::ConvTranspose2dOptions(128, 64, 3).padding(1).bias(false)),
        tnn::InstanceNorm2d(64),
        tnn::ReLU(true),
        tnn::ConvTranspose2d(tnn::ConvTranspose2dOptions(64, 32, 3).padding(1).bias(false)),
        tnn::InstanceNorm2d(32),
        tnn::ReLU(true),
        tnn::ConvTranspose2d(tnn::ConvTranspose2dOptions(32, 3, 9).stride(2).padding(4).bias(false)),
        tnn::Sigmoid());
    register_module("decoder_net", m_decoder);`

i think the question is forward function of ConvTranspose2d which have 2 parameters
`Tensor forward(const Tensor& input,
                 const c10::optional<at::IntArrayRef>& output_size = c10::nullopt);
`
when i call `m_decoder->forward(input);`, above issue occur. default parameter seems not work

**Comment by mruberry** [comment]
Got it. Thank you for clarifying!

**Comment by yf225** [comment]
@guijiyang Thanks for the report! This bug is fixed in https://github.com/pytorch/pytorch/pull/33027 and the fix is already available in our nightly builds. And it will be included in our 1.5 release as well (which is due in a few weeks).
